packages/sdk/react-client/src/echo/useQuery.ts

```diff
@@ -4,54 +4,56 @@
 
 import { useMemo, useSyncExternalStore } from 'react';
 
-import {
-  type Echo,
-  Filter,
-  type FilterSource,
-  type Query,
-  type QueryOptions,
-  type Space,
-  isSpace,
-} from '@dxos/client/echo';
-import type { BaseObject } from '@dxos/echo-schema';
+import { type Echo, Filter, type Live, Query, type Space, isSpace } from '@dxos/client/echo';
+
+// TODO(dmaretskyi): Queries are fully serializable, so we can remove `deps` argument.
+interface UseQueryFn {
```

Comment:
this type overrides the unknown I assume?

Reply:
yes